src/Controller.js

```diff
@@ -20,33 +23,38 @@ class ActivationSingleton {
 			this.lang = 'en';
 			this.translations = {};
 
-			ActivationSingleton.instance = this;
+			Controller.instance = this;
```

Comment:
I can't comment on lines that aren't part of the diff; but it looks like `this.$contentWrapper` and `this.$originalContent` are no longer being used (lines 21-22)?

Reply:
Good point, I moved these to the model.

Also, I think that we can get rid of the singleton signature... it's not really needed since we control the entire code and we're not actually creating a new instance, so there's no real need to fortify against ourselves.

I was trying to be careful how much I rewrite though, so we can do that as a followup.